_pages/dev/performance/core-web-vitals/largest-contentful-paint.md

```diff
@@ -0,0 +1,62 @@
+
+### Set high Fetch Priority for LCP image
+
+To improve the LCP metric, you can set the `fetchpriority` HTML attribute to `high` on your `<img>` elements that represent the Largest Contentful Paint image. It can be done in various ways, which is described in a separate document which we recommend to follow: [Prioritize Largest Contentful Paint image](./prioritize-largest-content-paint-image.md).
+
+### Lazy load all non-LCP media by default
+
+To prevent unnecessary images from competing for the Network bandwidth with the Largest Contentful Paint image, you can enable lazy loading by default of all images in Spartacus `<cx-media>` components. Thanks to this, the network bandwidth will be used only for necessary requests, but not for images that are outside of the initially visible part of the website. 
+
+Please do one of the following
+
+a) set the global Spartacus configuration `provideConfig({ imageLoadingStrategy: ImageLoadingStrategy.LAZY })`. 
+
+or
+
+b) alternatively, enable the feature toggle `lazyLoadImagesByDefault`, which is available since Spartacus v2211.43 (which under the hood does the same thing as the option a).
+
+### LCP image in won't and shouldn't be lazy loaded
+Please mind that the `<cx-media>` with the input `[fetchPriority]="'ImageFetchPriority.HIGH'"` (the feature that is described below) will not be lazy loaded despite the global config of lazy loading all images by default.This is a desired behavior to ensure that the Largest Contentful Paint image is loaded as soon as possible.
+
+### Page layout must be stable from the beginning to lazy-load images outside of the viewport
+For the browser to lazy-load images that are not in the viewport, the page layout needs to be stable from the beginning of displaying the page. Otherwise, all the images that initially are in the viewport (e.g due to having an initial height of `0px`) and are moved outside the initial viewport only after a delay, they might be all considered by the browser as candidates for eager loading, despite they being configured to be lazy loaded. It will make the lazy-loading ineffective. 
+
+To have a stable layout, please follow the Spartacus guide on keeping good [Cumulative Layout Shift (CLS)](./cumulative-layout-shift.md) metric.
+
+### Preconnect to the media domain unless it is the same as the storefront domain
+
+To further improve the LCP metric, Spartacus automatically creates `<link rel="preconnect" href="<MEDIA DOMAIN>">` based on the configured `mediaBaseUrl`, to perform the DNS and TLS requests as soon as possible, to later being able to fetch the images from that domain without additional roundtrips. This is supported since Spartacus v2211.43 when the feature toggle `createMediaPreconnectLink` is enabled.
+
+That said, it's even recommended to avoid the need to preconnect, i.e. it's recommended to serve images from **the same domain** as the storefront's domain, if only possible. Then no preconnecting to a media domain is needed.
```

Comment:
```suggestion
That said, it's even recommended to avoid the need to preconnect, i.e. it's recommended to serve images from **the same domain** as the storefront's domain, if only possible. Then no preconnection to a media domain is needed.
```